pkg/wsc110/keywords.go

```diff
@@ -2,6 +2,9 @@ package wsc110
 
 // Keywords in struct for repeatability
 type Keywords struct {
-	Keyword []string `xml:"ows:Keyword" yaml:"keyword"`
-	Type    string   `xml:"ows:Type,omitempty" yaml:"type"`
+	Keyword []string `xml:"ows:Keyword"`
+	Type    *struct {
+		Text      string  `xml:",chardata"`
```

Comment:
Meh... "text". Meh.

Reply:
??

Reply:
Het is wel expliciet, maar nu moet je altijd het text gebruiken om dit een waarde te geven. Daarom meh. Maar ik zie ook niet hoe je dat anders kan doen (zonder dat het weer ambigue wordt).